TestBenches/TrackletProcessor_test.cpp

```diff
@@ -96,6 +96,12 @@ int main()
 #else
   #error "Undefined seed"
 #endif
+#if SEED_ == L2D1_
```

Comment:
We should see if this can be done in a cleaner way (but should not hold up this PR - can yo enter as an issue?)

Reply:
I've just replaced all the NTEUnits with NCOPY instead, since NTEUNITs is no longer a template parameter of TP there was no need to use it to determine ncopy in the testbench.